packages/strapi-connector-bookshelf/lib/queries.js

```diff
@@ -59,12 +59,14 @@ module.exports = function createQueryBuilder({ model, strapi }) {
    */
   function find(params, populate, { transacting } = {}) {
     const filters = convertRestQueryParams(params);
+    const query = buildQuery({ model, filters });
 
     return model
-      .query(buildQuery({ model, filters }))
+      .query(query)
       .fetchAll({
         withRelated: populate,
         transacting,
+        publicationState: filters.publicationState,
```

Comment:
You've put that option here in order to retrieve it in the lifecycle?

Reply:
Yes, we use this in `populate.js` (`populateFetch`)